### PR TITLE
Update ru_RU.lang

### PR DESCRIPTION
**Pull request by xMrVizzy**
Just merge it @HenryLoenwind or @CrazyPants!

### CODE REVIEW

File: resources/assets/enderio/lang/ru_RU.lang [hunk 1]
```diff
@@ -165,13 +163,13 @@ fluid.liquid_sunshine=Жидкий солнечный свет
 tile.fluid.liquid_sunshine.name=Жидкий солнечный свет
 item.bucketLiquid_sunshine.name=Ведро жидкого солнечного света
 
-fluid.cloud_seed=Cloud Seed
-tile.fluid.cloud_seed.name=Cloud Seed
-item.bucketCloud_seed.name=Cloud Seed Bucket
+fluid.cloud_seed=Облако семян
```
Comment (AnrDaemon) [inline resources/assets/enderio/lang/ru_RU.lang]: Invalid translation, literally meaning "cloud of seeds".

File: resources/assets/enderio/lang/ru_RU.lang [hunk 2]
```diff
@@ -332,7 +330,7 @@ tile.blockCapBank.activated.name=Блок конденсаторов
 tile.blockCapBank.vibrant.name=Вибрирующий блок конденсаторов
 tile.blockCapBank.tooltip.detailed.line1=Мульти-блок для хранения энергии.
 tile.blockCapBank.tooltip.detailed.line2=Может заряжать RF-предметы.
-tile.blockCapBank.tooltip.detailed.line3=Shift-ПКМ, по экрану пустой рукой,
+tile.blockCapBank.tooltip.detailed.line3=SHIFT+ПКМ, по экрану пустой рукой,
```
Comment (AnrDaemon) [inline resources/assets/enderio/lang/ru_RU.lang]: "Shift" is a name of the key. Not an abbreviation.
Applies to the rest of occurences of this same name.

File: resources/assets/enderio/lang/ru_RU.lang [hunk 3]
```diff
@@ -557,9 +558,9 @@ enderio.gui.inventorypanel.tooltip.return.line5=Click to change into storage are
 enderio.gui.inventorypanel.tooltip.storage.line1=Область хранения предметов
 enderio.gui.inventorypanel.tooltip.storage.line2=Small temporary item storage area.
 enderio.gui.inventorypanel.tooltip.storage.line3=Click to change into return area
-enderio.gui.inventorypanel.tooltip.filterslot.line1=Filter card slot
-enderio.gui.inventorypanel.tooltip.filterslot.line2=Place configured item filters in
-enderio.gui.inventorypanel.tooltip.filterslot.line3=here to select which items to show.
+enderio.gui.inventorypanel.tooltip.filterslot.line1=Слот для фильтра
+enderio.gui.inventorypanel.tooltip.filterslot.line2=Поместите настроенный предметный фильтр,
+enderio.gui.inventorypanel.tooltip.filterslot.line3=чтобы выбрать, какие предметы будут отображаться.
```
Comment (AnrDaemon) [inline resources/assets/enderio/lang/ru_RU.lang]: Probably, "чтобы выбрать предметы для отображения." would be a better wording.
Length-wise.

File: resources/assets/enderio/lang/ru_RU.lang [hunk 4]
```diff
@@ -573,23 +574,23 @@ enderio.gui.inventorypanel.tooltip.sort.count_down.line2=Начиная с на
...
 enderio.gui.inventorypanel.tooltip.sort.count_up.line2=Начиная с одиночным количеством
 enderio.gui.inventorypanel.tooltip.refill.line1=Refills the crafting grid
-enderio.gui.inventorypanel.tooltip.refill.line2=A normal click puts items
-enderio.gui.inventorypanel.tooltip.refill.line3=for one crafting opperations,
-enderio.gui.inventorypanel.tooltip.refill.line4=while a shift click tries to
-enderio.gui.inventorypanel.tooltip.refill.line5=fill the crafting grid.
+enderio.gui.inventorypanel.tooltip.refill.line2=Обычное нажатие поместит предметы
+enderio.gui.inventorypanel.tooltip.refill.line3=для одной операции крафта,
+enderio.gui.inventorypanel.tooltip.refill.line4=в то время как при нажатии SHIFT попытается
+enderio.gui.inventorypanel.tooltip.refill.line5=заполнить сетку крафта.
```
Comment (AnrDaemon) [inline resources/assets/enderio/lang/ru_RU.lang]: Is that me, or the original text is not in English?

File: resources/assets/enderio/lang/ru_RU.lang [hunk 5]
```diff
@@ -573,23 +574,23 @@ enderio.gui.inventorypanel.tooltip.sort.count_down.line2=Начиная с на
...
 enderio.gui.inventorypanel.tooltip.fuelTank=Питательный резервуар
 enderio.gui.inventorypanel.tooltip.itemsstored=Хранится предметов: %s
-enderio.gui.inventorypanel.tooltip.addrecipe.line1=Store the current crafting recipe
-enderio.gui.inventorypanel.tooltip.addrecipe.line2=A maximum of 6 recipes can be stored.
-enderio.gui.inventorypanel.tooltip.addrecipe.line3=Left clicking a stored recipe will try to put
-enderio.gui.inventorypanel.tooltip.addrecipe.line4=items for one crafting operation into the grid.
-enderio.gui.inventorypanel.tooltip.addrecipe.line5=Shift left clicking tries to fill the crafting
-enderio.gui.inventorypanel.tooltip.addrecipe.line6=grid to the maximum.
+enderio.gui.inventorypanel.tooltip.addrecipe.line1=Сохраняет текущий рецепт крафта
+enderio.gui.inventorypanel.tooltip.addrecipe.line2=Максимум могут быть сохранены 6 рецептов.
```
Comment (AnrDaemon) [inline resources/assets/enderio/lang/ru_RU.lang]: "Максимум 6 рецептов может быть сохранено."
or
"Может быть сохранено максимум 6 рецептов."

"Максимум могут" - it isn't Russian to begin with.

File: resources/assets/enderio/lang/ru_RU.lang [hunk 6]
```diff
@@ -761,26 +762,26 @@ tile.blockPainter.tooltip.detailed.line6=и большинства машин
...
+item.itemYetaWrench.tooltip.detailed.line6=Используйте SHIFT+колёсико мыши или '%s', чтобы
 item.itemYetaWrench.tooltip.detailed.line7=отображать только один вид труб
 
 item.itemMagnet.name=Электромагнит
 item.itemMagnet.tooltip.detailed.line1=Тянет ближайшие к Вам предметы
-item.itemMagnet.tooltip.detailed.line2=Shift-ПКМ, чтобы включить/выключить
-item.itemMagnet.tooltip.detailed.line3=Должен быть запитан, активен и находиться на
+item.itemMagnet.tooltip.detailed.line2=SHIFT+ПКМ, чтобы включить/выключить
+item.itemMagnet.tooltip.detailed.line3=Должен быть запитан энергией, активен и находиться на
 item.itemMagnet.tooltip.detailed.line4=панели быстрого доступа (или в слоте Baubles) для функционирования
```
Comment (AnrDaemon) [inline resources/assets/enderio/lang/ru_RU.lang]: -запитан энергией
+заряжен
(i.e. "charged")

File: resources/assets/enderio/lang/ru_RU.lang [hunk 7]
```diff
@@ -817,23 +818,23 @@ item.darkSteel_shears.name=Тёмные ножницы
...
 
-enderio.item.darkSteel_pickaxe.tooltip.effPowered=Эффективность при питании
+enderio.item.darkSteel_pickaxe.tooltip.effPowered=Эффективность при заряде энергии
 enderio.item.darkSteel_pickaxe.tooltip.effObs=Эффективность уничтожения обсидиана
 enderio.item.darkSteel_pickaxe.tooltip.cost=стоит
 
 enderio.item.darkSteel_axe.tooltip.multiHarvest=Присядьте, чтобы срубить несколько брёвен
 
-enderio.item.darkSteel_shears.tooltip.multiHarvest=Стригут несколько блоков/овец, если запитаны
+enderio.item.darkSteel_shears.tooltip.multiHarvest=Стригут несколько блоков/овец, если запитаны энергией
```
Comment (AnrDaemon) [inline resources/assets/enderio/lang/ru_RU.lang]: -запитаны
+заряжены

File: resources/assets/enderio/lang/ru_RU.lang [hunk 8]
```diff
@@ -1251,14 +1252,62 @@ enderio.tooltip.uid=UID:
...
+enderio.config.capacitor.alloy_smelter_power_use=Завод сплавов: Power use per tick (будет масштабироваться при использовании конденсатора)
+enderio.config.capacitor.buffer_power_buffer=Энергетический/универсальный буфер: Размер внутреннего буфера энергии (будет масштабироваться при использовании конденсатора)
+enderio.config.capacitor.buffer_power_intake=Энергетический/универсальный буфер: Максимальная энергия, потребляемая за тик (будет масштабироваться при использовании конденсатора)
+enderio.config.capacitor.crafter_power_buffer=Создатель: Размер внутреннего буфера энергии (будет масштабироваться при использовании конденсатора)
+enderio.config.capacitor.crafter_power_intake=Создатель: Максимальная энергия, потребляемая за тик (будет масштабироваться при использовании конденсатора)
+enderio.config.capacitor.crafter_ticks=Создатель: Тиков за операцию крафта (будет масштабироваться при использовании конденсатора)
 
-// There's a 1% change that a soul vial containing a pig is names this:
+// There's a 1% change that a soul vial containing a pig is named this:
```
Comment (AnrDaemon) [inline resources/assets/enderio/lang/ru_RU.lang]: Split this change into a separate PR. This is a typo in the original code.

File: resources/assets/enderio/lang/ru_RU.lang (hunk 1 above)
Comment (xMrVizzy) [inline resources/assets/enderio/lang/ru_RU.lang]: Suggest your version

File: resources/assets/enderio/lang/ru_RU.lang (hunk 2 above)
Comment (xMrVizzy) [inline resources/assets/enderio/lang/ru_RU.lang]: I know it, it i fixed in commit but not uploade

File: resources/assets/enderio/lang/ru_RU.lang (hunk 3 above)
Comment (xMrVizzy) [inline resources/assets/enderio/lang/ru_RU.lang]: here to select which items to show, "here to select" - чтобы выбрать, "which items to show" - какие предметы будут отображаться, but your "чтобы выбрать предметы для отображения." too cool

File: resources/assets/enderio/lang/ru_RU.lang (hunk 4 above)
Comment (xMrVizzy) [inline resources/assets/enderio/lang/ru_RU.lang]: It that you!

File: resources/assets/enderio/lang/ru_RU.lang (hunk 5 above)
Comment (xMrVizzy) [inline resources/assets/enderio/lang/ru_RU.lang]: Yes, in this you are exactly right!

File: resources/assets/enderio/lang/ru_RU.lang (hunk 6 above)
Comment (xMrVizzy) [inline resources/assets/enderio/lang/ru_RU.lang]: It's translate from previous tranlsator "-запитан энергией"

File: resources/assets/enderio/lang/ru_RU.lang (hunk 7 above)
Comment (xMrVizzy) [inline resources/assets/enderio/lang/ru_RU.lang]: It's translate from previous tranlsator "-запитан энергией"

File: resources/assets/enderio/lang/ru_RU.lang (hunk 6 above)
Comment (AnrDaemon) [inline resources/assets/enderio/lang/ru_RU.lang]: You are supposed to translate the _meaning_ of the _original_ text to the adequate native wording.
Translating separate words, or editing someone else's translation without checking the original source often leads to the misinterpretation and ends in a wrong translation.

File: resources/assets/enderio/lang/ru_RU.lang (hunk 4 above)
Comment (HenryLoenwind) [inline resources/assets/enderio/lang/ru_RU.lang]: No, original text not nice sentence in English either. Maybe some of our native speakers could have a look?

Comment (tterrag1098) [inline resources/assets/enderio/lang/ru_RU.lang]: puts -> places
operations -> operation

Just those two little changes would improve the readability a lot

Comment (bookerthegeek) [inline resources/assets/enderio/lang/ru_RU.lang]: While slightly off it is correct. We could change it around to be clearer if it is deemed confusing. (The English part)

File: resources/assets/enderio/lang/ru_RU.lang (hunk 8 above)
Comment (HenryLoenwind) [inline resources/assets/enderio/lang/ru_RU.lang]: Didn't I fix that one already?

However, I don't care about comments bein in sync between the languages. In fact, it's a bad thing. Put your own comments in. If you had to look up how a string is used in-game, write it down for the next person to work on the translation. If you translated some word oddly, write down why (e.g. same wording as OtherMod).

And BTW, to check if your translation is complete, or has dead lines, run "gradle checkTranslations"

Comment (tterrag1098) [inline resources/assets/enderio/lang/ru_RU.lang]: This isn't even how comments work in lang files. They are escaped with # not //

File: resources/assets/enderio/lang/ru_RU.lang (hunk 4 above)
Comment (AnrDaemon) [inline resources/assets/enderio/lang/ru_RU.lang]: With changes suggested by @tterrag1098 , it is much clearer (or at least more correct).
But the part that still confusing me is "fill the crafting grid".

**Comment by xMrVizzy** [comment]
@HenryLoenwind how can i enable it "gradle checkTranslations"? Where?

**Comment by AnrDaemon** [comment]
http://lmgtfy.com/?q=minecraft+gradle

**Comment by xMrVizzy** [comment]
@AnrDaemon напиши пожалуйста как это сделать через Гит Клиент?

**Comment by mindforger** [comment]
@xMrVizzy this is noting that has to be done with the git client, gradle is a separate tool, download this actual git project and setup your gradle using the gradlew.bat like found on the internet and then run gradlew checkTranslations

File: resources/assets/enderio/lang/ru_RU.lang (hunk 4 above)
Comment (bookerthegeek) [inline resources/assets/enderio/lang/ru_RU.lang]: Change it to "attempts up to 64 operations"?

**Comment by xMrVizzy** [comment]
@AnrDaemon можешь объяснить по-русски как это сделать? Я так то моды пишу но с этой системой Travis Ci не разбирался

**Comment by AnrDaemon** [comment]
Я сам с gradle никогда не работал. Извини.
Если будут вопросы по гайдам, пиши, помогу перевести.

**Comment by xMrVizzy** [comment]
Хорошо.

**Comment by xMrVizzy** [comment]
@mindforger can you help me with it please :)
